db/migrations/sqls/20220426160810-init-up.sql

```diff
@@ -0,0 +1,504 @@
+CREATE TABLE IF NOT EXISTS block
```

Comment:
@AndreiNavoichyk,
do we plan to update this file if the database schema will change in the original forked repo?

Reply:
@upalinski, we can import the original `backend.sql` file into this migration script.